auslib/web/common/rules_spec.yml

```diff
@@ -0,0 +1,124 @@
+paths:
+  /rules:
```

Comment:
I got to thinking about this API a bit more and I think we should make it a bit more defined, and versioned. How about putting all of the new endpoints behind: /api/v1?